### ISSUE TITLE
What constitutes a "stack trace"?

### ISSUE THREAD

**Issue by zeke** [capaj/hyperlinks#3]
It looks like this is the current pattern:

``` js
const stackTraceRe = /\((\/.*:\d*:\d*)\)/g
```

Why are the opening and closing parentheses necessary?

It would be cool if this would match errors from `standard`. An example:

```
❯ standard

standard: Use JavaScript Standard Style (http://standardjs.com)
  /Users/zeke/electron/electron-api-docs/index.js:1:1: 'wrong' is not defined.
```

**Comment by garrows** [comment]
It's not finding my stack traces either. Here is an example:

```
Uncaught AssertionError: expected 0 to equal 1
+ expected - actual

+1
-0

at Assertion.fail (/Users/glen/redacted/node_modules/should/cjs/should.js:205:17)
at Assertion.Object.defineProperty.value (/Users/glen/redacted/node_modules/should/cjs/should.js:277:19)
at Request._callback (/Users/glen/redacted/test/specs/redacted.js:40:33)
at Request.self.callback (/Users/glen/redacted/node_modules/request/request.js:187:22)
at emitTwo (events.js:87:13)
at Request.emit (events.js:172:7)
at Request.<anonymous> (/Users/glen/redacted/node_modules/request/request.js:1048:10)
at emitOne (events.js:77:13)
at Request.emit (events.js:169:7)
at IncomingMessage.<anonymous> (/Users/glen/redacted/node_modules/request/request.js:969:12)
at emitNone (events.js:72:20)
at IncomingMessage.emit (events.js:166:7)
at endReadableNT (_stream_readable.js:905:12)
at doNTCallback2 (node.js:441:9)
at process._tickCallback (node.js:355:17)
```

Thanks in advance if you get it working.

**Comment by capaj** [comment]
@zeke probably not the cleanest way, but I've resolved this issue by adding another regex which matches standard output. Let me know if it doesn't work for you.

released as 0.2.6
